app/src/main/kotlin/com/wire/android/ui/debug/DebugDataOptions.kt

```diff
@@ -641,20 +437,41 @@ private fun DebugToolsOptions(
                 )
             }
         )
-        RowItemTemplate(
-            modifier = Modifier.wrapContentWidth(),
-            title = {
-                Text(
-                    style = MaterialTheme.wireTypography.body01,
-                    color = MaterialTheme.wireColorScheme.onBackground,
-                    text = prettyPrintMap(dependenciesMap),
-                    modifier = Modifier.padding(start = dimensions().spacing8x)
-                )
-            }
-        )
     }
 }
 
+/**
+ *
+ */
+@Composable
+fun dependenciesItem() {
+    val applicationContext = LocalContext.current.applicationContext
+    val state: MutableState<String> = remember { mutableStateOf(String.EMPTY) }
+
+    LaunchedEffect(key1 = Unit) {
+        withContext(Dispatchers.IO) {
+            val title = applicationContext.resources.getString(R.string.item_dependencies_title)
+            applicationContext.getDependenciesVersion().let {
+                prettyPrintMap(it, title)
+            }.also {
+                withContext(Dispatchers.Main) {
+                    state.value = it
+                }
+            }
+        }
+    }
```

Comment:
Question: Can this be moved to the view model, we already have the `ApplicationContext` in there. So we can do it there instead of launching this Effect